spec/spec_helper.rb

```diff
@@ -11,6 +31,6 @@
   config.shared_context_metadata_behavior = :apply_to_host_groups
 end
 
-def sign_in(user)
-  session[:user_id] = user.id
+def sign_in(user) 
+  page.set_rack_session(user_id: user.id)
```

Comment:
To remove some duplication, I typically like to make this method such that you can just pass it a symbol (perhaps a factory trait) or a User instance, and it will take care of creating a user for you if necessary, Cf. https://github.com/umts/umasstransit.rodeo/blob/6feb613407a9a04caa434665f2eab525184e57c6/spec/spec_helper.rb#L23-L35